falcano/model.py

```diff
@@ -750,35 +749,23 @@ def update(self, actions, condition=None):
             raise TypeError('the value of `actions` is expected to be a non-empty list')
 
         kwargs = {
-            RETURN_VALUES: ALL_NEW,
+            'return_values': ALL_NEW,
```

Comment:
Use the constants imported. If they have to be lowercase, you can do
```suggestion
            strincase.snake_case(RETURN_VALUES): ALL_NEW,
```

Reply:
Nice... want me to do this everywhere?

Reply:
that'd be nice!